### PR TITLE
Fixed for CUST-2021,2022,2023

### PR DESCRIPTION
**Pull request by rwrzesien**
This implements:
https://whentomanage.atlassian.net/browse/CUST-2021
https://whentomanage.atlassian.net/browse/CUST-2022
https://whentomanage.atlassian.net/browse/CUST-2023

QA verified already in tickets:

### CODE REVIEW

File: client/app/material/md-switch.scss [hunk 1]
```diff
@@ -1,15 +1,57 @@
 md-switch {
-  &.md-primary {
+
+  &:not([disabled]) {
+
+    .md-bar {
+      background-color: rgba(214,214,214,0.5);
```
Comment (jsmylnycky) [inline client/app/material/md-switch.scss]: These colors should all be in variables.scss

**Comment by rwrzesien** [comment]
@jsmylnycky Updated.

**Comment by jsmylnycky** [comment]
:+1: You can merge squash

**Comment by rwrzesien** [comment]
@jsmylnycky Merged and pushed, commit:
90c1f38c1539113959a84fa6609c6b0f58189f67

**Comment by jsmylnycky** [comment]
Released v0.3.6

**Pull request closed without merging**